testing_app/test/favorites_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:testing_app/models/favorites.dart';
+import 'package:testing_app/screens/favorites.dart';
+
+Favorites favoritesList;
+Widget createFavoritesScreen() => ChangeNotifierProvider<Favorites>(
+      create: (context) {
+        favoritesList = Favorites();
+        return favoritesList;
+      },
+      child: MaterialApp(
+        home: FavoritesPage(),
+      ),
+    );
+void addRandomItems() {
+  for (var i = 0; i < 10; i += 2) {
+    favoritesList.add(Random().nextInt(50));
+  }
+}
```

Comment:
This Screen is dependent on the Model for getting data. Since we are testing it directly, it was empty. So, I did something like this to add data into the model. Is this okay?